docker/Dockerfile

```diff
@@ -49,4 +49,4 @@ WORKDIR /kms
 
 ARG CCF_PLATFORM
 ENV CCF_PLATFORM=${CCF_PLATFORM}
-CMD ["/bin/bash", "-c", "make start-host-idp & (./scripts/kms_wait.sh && make setup && tail -f /dev/null)"]
+CMD ["/bin/bash", "-c", "source .venv_ccf_sandbox/bin/activate && make start-host-idp & (./scripts/kms_wait.sh && make setup && tail -f /dev/null)"]
```

Comment:
Why are we doing this?

`make start-host-idp` calls `make build` which calls `./scripts/set_python_env.sh` which activates the virtual environment

Reply:
Resolved by merging main in